products/sle/main.pm

```diff
@@ -5,6 +5,13 @@ use lockapi;
 use autotest;
 use needle;
 use File::Find;
+use File::Basename;
+
+BEGIN {
+    # add search path for lib/utils.pm
+    unshift @INC, dirname(__FILE__) . '/../../lib';
+}
+use utils;
```

Comment:
yes, but that would require a new deployment. We can do it in both places for the time being and remove later